packages/react/ds-core-form/biome.json

```diff
@@ -1,7 +1,7 @@
 {
-  "$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
+  "$schema": "https://biomejs.dev/schemas/2.0.0/schema.json",
   "extends": ["@canonical/biome-config"],
   "files": {
-    "include": ["src", "*.json"]
+    "includes": ["src", "*.json", "!public"]
```

Comment:
Looks like a functional change in what I would expect would be a configuration-only PR - could you explain the change to `public`?